db/migrate/20160426232823_change_required_fields.rb

```diff
@@ -0,0 +1,21 @@
+class ChangeRequiredFields < ActiveRecord::Migration
+  def change
```

Comment:
It might be better to migrate one table at a time.  That way if you have to roll back, you won't have to roll back all the tables.